src/components/AppBar/NavBar.style.ts

```diff
@@ -142,9 +144,18 @@ export const IconButton = styled(MuiIconButton).attrs(props => ({
   disableRipple: true,
   disableFocusRipple: true,
 }))`
-  &:focus {
+  ${fonts.regularBookMidWeight};
```

Comment:
Also here, I think that ideally we want to use theme variables by accessing the theme context directly instead of importing the variable

```tsx
export const IconButton = styled(MuiIconButton).attrs(props => ({ /* ... */ })`
  ${props => props.theme.fonts.regularBookMidWeight};
`;
```

Reply:
ah yes good catch